src/app/cube/details/details.component.ts

```diff
@@ -335,6 +352,35 @@ export class DetailsComponent implements OnInit, OnDestroy {
     }
   }
 
+  /**
+   * Opens the Change Log modal for a specified learning object and fetches its change logs
+   */
+  async openViewAllChangelogsModal() {
+    if (!this.openChangelogModal ) {
+      this.openChangelogModal = true;
+      this.loadingChangelogs = true;
+      this.changelogLearningObject = this.learningObject;
+      try {
+        this.changelogs = await this.changelogService.fetchAllChangelogs({
```

Comment:
This function needs to be extended to take revisions into account. The Learning Object service endpoint accepts a `minusRevision` query parameter. The `fetchAllChangelogs` function will need that parameter to know how many changelogs to return to the client.

- If a user is viewing changelogs for a released copy of a Learning Object on the details page, `minusRevision` should be set to true. This tells the service to only return changelogs that are relevant to the released copy. 

- If a user is viewing changelogs for a revision (working copy) of a Learning Object on the details page, `minusRevision` should be set to false. This tells the service to return all changelogs that are relevant to the released copy and the working copy of the Learning Object.